dumper/app/src/main/resources/oracle-stats/awr/hist-cmd-types-awr.sql

```diff
@@ -0,0 +1,46 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  A.con_id "ConId",
+  B.command_type "CommandType",
+  to_char(C.begin_interval_time, 'hh24') "Hour",
+  D.name "CommandName",
+  count(1) "Count",
+  sum(buffer_gets_delta) "SumBufferGets",
```

Comment:
Can we add _total columns to the result query as well? Total columns are more important for us because they deliver information about the whole query performance.

_total: This column shows the cumulative value for the metric since the last time the statistics were reset. For example, cpu_time_total would indicate the total CPU time consumed by a SQL statement throughout its lifetime.
_delta: This column shows the change in the metric's value between the current and previous snapshot. For instance, cpu_time_delta would reflect the CPU time used by a statement since the last time the statistics were collected.

Reply:
Done, I added the '_total' columns